mapentity/views/mixins.py

```diff
@@ -120,8 +121,15 @@ def __init__(self):
             class filterklass(MapEntityFilterSet):
                 class Meta:
                     model = _model
-                    fields = [field.name for field in _model._meta.get_fields() if
-                              not isinstance(field, GeometryField) and not isinstance(field, GenericRelation)]
+                    fields = [
+                        field.name
+                        for field in _model._meta.get_fields()
+                        if not isinstance(field, GeometryField)
+                        and not isinstance(field, GenericRelation)
+                        and not isinstance(field, GenericForeignKey)
+                        and not isinstance(field, GenericRel)
+                        and not isinstance(field, FileField)
```

Comment:
you can use a single tuple for this isinstance(field, (GeometryField, GenericRelation, xxx))

https://docs.python.org/3/library/functions.html#isinstance

Reply:
what is GenericRel ?

Reply:
Thanks for the refactor suggestion

GenericRel is the reverse for GenericRelation (which is supposed to represent the reverse for GenericForeignKey but actually also has their own reverse)

So this field on Trek
`view_points = GenericRelation('common.HDViewPoint', related_query_name='trek')`

means field `HDVIewPoint.trek` exists which then raises

```
File "/opt/venv/lib/python3.8/site-packages/django_filters/filterset.py", line 373, in filter_for_field
    assert filter_class is not None, (
AssertionError: filterklass resolved field 'trek' with 'exact' lookup to an unrecognized field type GenericRel. Try adding an override to 'Meta.filter_overrides'. See: https://django-filter.readthedocs.io/en/main/ref/filterset.html#customise-filter-generation-with-filter-overrides
```